locales/en.default.schema.json

```
@@ -1978,6 +1978,162 @@
       "presets": {
         "name": "Video"
       }
+    },
+    "slideshow": {
+      "name": "Slideshow",
+      "settings": {
+        "layout": {
+          "label": "Layout",
+          "options__1": {
+            "label": "Full width"
+          },
+          "options__2": {
+            "label": "Grid"
+          },
+          "options__3": {
+            "label": "Grid and peek"
+          }
+        },
+        "slide_height": {
+          "label": "Slide height",
+          "options__1": {
+            "label": "Adapt to first image"
+          },
+          "options__2": {
+            "label": "Small"
+          },
+          "options__3": {
+            "label": "Medium"
+          },
+          "options__4": {
+            "label": "Large"
+          }
+        },
+        "slider_visual": {
+          "label": "Slider visual",
+          "options__1": {
+            "label": "Numbers"
+          },
+          "options__2": {
+            "label": "Dots"
+          },
+          "options__3": {
+            "label": "Counter"
```

```suggestion
          "options__1": {
            "label": "Counter"
          },
          "options__2": {
            "label": "Dots"
          },
          "options__3": {
            "label": "Numbers"
```

Changing it for alphabetical order.